package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amplia-client",
-  "version": "1.3.2",
```

Comment:
Please, only change the version of the package when you publish the library, after reviewing this PR and merging it into the develop.